frontend/src/services/fetch/fetchers/clientApplicationFetcher.ts

```diff
@@ -0,0 +1,22 @@
+"use client";
+
+interface ClientApplicationStartResponse {
+  message: string;
+  applicationId: string;
+}
+
+export const startApplication = async (
+  token?: string,
+): Promise<ClientApplicationStartResponse> => {
+  if (!token) {
+    throw new Error(`Error starting application`);
+  }
+  const res = await fetch("/api/applications/start", {
```

Comment:
it looks like this endpoint is returning an application id, which is then being passed as in the id path segment for /formPrototype/:id, but the page there is expecting a competition id. Is that why things are 404ing, and what will be fixed in the next ticket?